### PR TITLE
Added program to get angle between clock hands

### PR DESCRIPTION
**Pull request by verdantfire**
### **Describe your change:**
The function will return the angle between the hour and minute hands for an input time in hours and minutes

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by itsvinayak** [comment]
Hey @verdantfire codespell fails, please have a look

File: other/angle_between_clock_hands.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+time in hours and minutes
+
+>>> angle_between_hands(12,30)
+165.0
+>>> angle_between_hands(4,45)
+127.5
+>>> angle_between_hands(13,59)
+Traceback (most recent call last):
+...
+ValueError: Hour value out of bounds (0-12)
```
Comment (dhruvmanila) [inline other/angle_between_clock_hands.py:12]: Please put the `doctest` in function docstring.

File: other/angle_between_clock_hands.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Problem description: https://en.wikipedia.org/wiki/Clock_angle_problem
+"""
+
+
+def angle_between_hands(hour: int, minutes: int) -> int:
+
+    # Raises error if hour value is not in the correct range
+    if not 0 <= hour <= 12:
+        raise ValueError("Hour value out of bounds (0-12)")
```
Comment (dhruvmanila) [inline other/angle_between_clock_hands.py:22]:
```suggestion
        raise ValueError("Hour value out of bounds [0-12]")
```
Square brackets are used to indicate inclusiveness and round brackets are used for exclusiveness.

File: other/angle_between_clock_hands.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    # Raises error if minutes value is not in the correct range
+    if not 0 <= minutes < 60:
+        raise ValueError("Minutes value out of bounds (0-59)")
```
Comment (dhruvmanila) [inline other/angle_between_clock_hands.py:26]:
```suggestion
        raise ValueError("Minutes value out of bounds [0-59]")
```
Same as above.

**Review by dhruvmanila — CHANGES_REQUESTED** [review]

**Pull request closed without merging**